docs/source/debugging.rst

```diff
@@ -0,0 +1,50 @@
+Debugging Errors
+================
+
+First, thank you for using Starfish and SpaceTx-Format! Feedback you provide on features and the
+user experience is critical to making Starfish a successful tool. Because we iterate quickly on this
+feedback to add new features, things change often, which can result in your code getting out of sync
+with your data. When that happens, you may observe errors.
+
+Most of the time, you can fix this problem by pulling the most recent version of the code,
+reinstalling starfish, and restarting your environment. If you're using starfish with datasets from
+spaceTx located on our cloudfront distribution, we're committed to keeping that data up to date.
+You can find it in a versioned folder, and the version will correspond to the version of starfish
+that it matches.
+
+If you're using your own data with starfish, you may need to re-run your data ingestion workflow
+based on TileFetcher and FetchedTile to generate up-to-date versions of spaceTx-format.
+
+Upgrading to a new version
+--------------------------
+
+If you've installed from pypi, upgrading is as simple as reinstalling starfish.
+
+.. code-block:: bash
+
+    pip3 install --upgrade starfish
```

Comment:
If they're using the virtualenv they setup with the starfish setup instrucitons I think this would just be  pip install --upgrade starfish, we should probably be consistent on our setup and debugging instructions to reflect one or the other.

Reply:
Reasonable -- I think both work inside the virtualenv, but I've edited this to be consistent.